src/api/v1/serializers/onbase-serializer.js

```diff
@@ -60,4 +65,43 @@ const serializeAdmission = (rawRows, osuId) => {
   ).serialize(rawAdmission);
 };
 
-export { serializeAdmission };
+/**
+ * A function to serialize raw financial aid data
+ *
+ * @param {object[]} rawRows Raw data rows from data source
+ * @param {string} osuId OSU ID
+ * @returns {object} Serialized resources data
+ */
+const serializeFinancialAid = (rawRows, osuId) => {
+  const rawFinancialAid = {
+    osuId,
+    type: 'financialAid',
```

Comment:
Is this string different from `financialAidResourceType`?

Reply:
Good catch, fixed.